ramp4-pcar4/ramp4-pcar4
# Fix panels sometimes not rendering at specific widths

### Related Item(s)
#1787

### Changes
- When there is only space for one panel, a newly opened panel will take priority over a currently pinned panel.

### Testing
Steps:
1. Open a panel and pin it (skip if there is already an opened and pinned panel there).
2. Make the screen width 576px to 831px.
3. Try opening another panel, such as the wizard.
4. Notice that the new panel opens.
